docs/api/protocol-ns.md

```diff
@@ -0,0 +1,309 @@
+# protocolNS
```

Comment:
```suggestion
# `NetworkService` Protocol
```

Reply:
The documentation linter and generator use this title as object name, spaces and special characters can not be used here.